### PR TITLE
C24_WMDE_Mobile_DE_05

### PR DESCRIPTION
**Pull request by moiikana**
- based on de 04 ctrl
- var has additional "thank you" text box

https://phabricator.wikimedia.org/T374103

### CODE REVIEW

File: src/components/ThankYouBox/ThankYouBox.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@mixin layout() {
+	.thank-you-container {
+		height: 60px;
```
Comment (Abban) [inline src/components/ThankYouBox/ThankYouBox.scss:3]: The height in the design is 40px

File: src/themes/Mikings/ThankYouBox/ThankYouBox.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@use 'src/components/ThankYouBox/ThankYouBox';
+
+.wmde-banner {
+	@include ThankYouBox.layout;
+
+	.thank-you-container {
+		font-weight: bold;
+		background: #ebe2fc;
```
Comment (Abban) [inline src/themes/Mikings/ThankYouBox/ThankYouBox.scss]: This is another thing we should create color variables for, and a flag I think.

But it also has me thinking, what do we do if they want to test 2 different colours of the box? We should chat about it in the daily tomorrow.

File: src/components/Icons/HeartIcon.vue [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<template>
+	<svg width="16" height="16" viewBox="0 0 16 16" fill="#6b4ba1" xmlns="http://www.w3.org/2000/svg">
```
Comment (Abban) [inline src/components/Icons/HeartIcon.vue]: When you put the colours into the swatch you can remove `fill="#6b4ba1"` and add `style="var( --thank-you-heart-fill )"`

**Review by Abban — CHANGES_REQUESTED** [review]
It all looks pretty good, but I'm hitting request changes to remind us to talk about the colour variables in the daily tomorrow.

File: src/components/ThankYouBox/ThankYouBox.scss (hunk 1 above)
Comment (moiikana) [reply src/components/ThankYouBox/ThankYouBox.scss:3]: the design in the figma file is overall a bit different to our actual current live banners.
In figma, both, submit button and thank you box are 40px high.
In our live banners, the submit button has a height of 60px though.
That's why I assume I should keep them the same height (60px).

Comment (Abban) [reply src/components/ThankYouBox/ThankYouBox.scss:3]: Ah right, grand so!

**Review by Abban — APPROVED** [review]

**Comment by moiikana** [comment]
(rebased because of package-lock file)

**Pull request merged**